typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -38,16 +41,16 @@ export interface ChainMetadata {
   publicRpcUrls: Array<{
     http: string;
     webSocket?: string;
-    pagination?: RpcPagination;
-    retry?: RetryOptions;
+    pagination?: RpcPaginationOptions;
+    retry?: RetryProviderOptions;
```

Comment:
I like that this is overridable, but IMO we should not expect developers to know how to override this, so I would personally suggest a default that does retry, maybe like once or twice with a second in between?

Reply:
Yep that's how it'll work with MultiProvider after this